haskell/src/Monocle/Client.hs

```diff
@@ -89,7 +93,11 @@ monocleReq path MonocleClient {..} body =
     initRequest <- parseUrlThrow (T.unpack $ baseUrl <> path)
     let request =
           initRequest
-            { requestHeaders = [("Accept", "*/*"), ("Content-Type", "application/json")],
+            { requestHeaders =
+                [ ("Accept", "*/*"),
+                  ("Content-Type", "application/json"),
+                  authorizationH
+                ],
```

Comment:
not sure how that works, perhaps it would be easier to mappend a list, e.g.:

```suggestion
                  ("Content-Type", "application/json")
                ] <> Data.Maybe.maybeToList authorizationM,
```

Reply:
maybeToList is useful ! right